service-loadbalancer/service_loadbalancer.go

```diff
@@ -277,6 +301,13 @@ func (cfg *loadBalancerConfig) write(services map[string][]service, dryRun bool)
 	conf["startSyslog"] = strconv.FormatBool(cfg.startSyslog)
 	conf["services"] = services
 
+	conf["sslCert"] = ""
+	if cfg.sslCert != "" {
+		conf["sslCert"] = "crt " + cfg.sslCert
+	}
+	if cfg.sslCaCert != "" {
+		conf["sslCert"] = conf["sslCert"].(string) + " ca-file " + cfg.sslCaCert
```

Comment:
can you avoid the assertion by constructing the string first and then just assigning it to sslCert?